misk-actions/src/main/kotlin/misk/web/interceptors/LogRequestResponse.kt

```diff
@@ -23,9 +23,7 @@ package misk.web.interceptors
 @Retention(AnnotationRetention.RUNTIME)
 @Target(AnnotationTarget.FUNCTION, AnnotationTarget.CLASS)
 annotation class LogRequestResponse(
-  // TODO(isabel): Remove sampling and includeBody. Will be replaced with rate limiting
-  val sampling: Double,
-  val includeBody: Boolean,
```

Comment:
To make the migration easier, I would recommend keeping the `sampling` and `includeBody` params so the old code still compile. They just won't do anything